src/Promotion.tsx

```diff
@@ -34,6 +36,7 @@ export const Promotion: React.FC<PromotionProps> = (props) => {
         })
         .catch(error => {
           console.error(error);
+          addError(error.errors);
```

Comment:
Do we still need to call setErrors if we're displaying our own toast?

Reply:
@shaunmaharaj,
I think we should display our own toast.
That's actually what I told before. Should we use the state of a component or our own toast for displaying messages (the same is in address form, login form, and registration form)?